app/(app)/[username]/[slug]/page.tsx

```
@@ -0,0 +1,685 @@
+                </>
+              )}
+            </div>
+
+            {/* Article content */}
+            <div className="prose mx-auto max-w-none dark:prose-invert lg:prose-lg">
+              {!isTiptapContent && <h1>{userPost.title}</h1>}
+
+              {isTiptapContent ? (
+                <div
+                  dangerouslySetInnerHTML={{
+                    __html: renderedContent ?? <NotFound />,
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)